source/accessibility/index.html.md.erb

```diff
@@ -47,21 +49,30 @@ The Equality and Human Rights Commission (EHRC) is responsible for enforcing the
 
 ##### Compliance status
 
-This website is fully compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard.
+This website is partially compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard.
 
 ### Preparation of this accessibility statement
 
-This statement was prepared on 2 September 2020. It was last reviewed on 29 March 2021.
+This statement was prepared on 2 September 2020. It was last reviewed on 18 June 2021.
 
-This website was last tested in March 2021.
+This website was last tested in April 2021.
 
 ## Using the Technical Documentation Template for your own documentation
 
-This [Technical Documentation Template](https://github.com/alphagov/tech-docs-template) was last tested in March 2021.
+This [Technical Documentation Template](https://github.com/alphagov/tech-docs-template) was last tested in April 2021. The test was carried out by the [Digital Accessibility Centre (DAC)](https://digitalaccessibilitycentre.org/).
+
+DAC tested the [GovWifi website](https://admin.wifi.service.gov.uk/users/sign_in) and the [GovWifi technical documentation](https://docs.wifi.service.gov.uk/).
+
+This Technical Documentation Template is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard, due to the non-compliances listed below:
+
+- On page load, the main page heading in the content is automatically focused. Keyboard-only users expect the focus to be at the top of the page and then move through the navigation elements. Keyboard-only users may therefore miss the navigation elements which would be confusing. This fails [WCAG 2.1 success criterion 2.4.3 - Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html).
+- The navigation elements are not marked-up as hierarchical headings. As screen reader users rely on correct semantic mark-up to determine relationships within the page, this makes navigating a page using a screen reader more difficult. This fails [WCAG 2.1 success criterion 1.3.1 - Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html).
+- There are sections of content with show / hide functionality. The `aria-expanded` attribute is not implemented correctly, so users may miss that menu items are expandable. This fails [WCAG 2.1 success criterion 4.1.2 - Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html).
+- The main page region is hidden from assistive technology when the search results dialog is triggered. This fails [WCAG 2.1 success criterion 3.2.2 - On Input](https://www.w3.org/WAI/WCAG21/Understanding/on-input.html) and [WCAG 2.1 success criterion 2.4.3 - Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html).
```

Comment:
```suggestion
- When a user searches, the search results dialogue hides the main page region from assistive technology. This fails [WCAG 2.1 success criterion 3.2.2 - On Input](https://www.w3.org/WAI/WCAG21/Understanding/on-input.html) and [WCAG 2.1 success criterion 2.4.3 - Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html).
```

Reply:
Suggested some tweaks to shorten the bullets and change passive to active voice.